apps/web/playwright/integrations-stripe.e2e.ts

```diff
@@ -69,9 +69,47 @@ test.describe("Stripe integration", () => {
     await expect(page.getByText("Unconfirmed")).toBeVisible();
     await expect(page.getByText("Pending payment").last()).toBeVisible();
   });
+
+  test("Paid booking should be able to be rescheduled", async ({ page, users }) => {
+    const user = await users.create();
+    const eventType = user.eventTypes.find((e) => e.slug === "paid") as Prisma.EventType;
+    await user.apiLogin();
+    await page.goto("/apps/installed");
+
+    await user.getPaymentCredential();
+    await user.setupEventWithPrice(eventType);
+    await user.bookAndPaidEvent(eventType);
+
```

Comment:
here is missing the part of check if eventType was paid with a stripe transaction.

Reply:
I am first booking a paid event and then trying to reschedule. So i just followed first steps from Can book a paid booking test in the same file.